sdk/rust-examples/shamir-secret-sharing/Xargo.toml

```diff
@@ -0,0 +1,5 @@
+[target.wasm32-arm-veracruz.dependencies]
+std = {}
+
+[patch.crates-io]
+compiler_builtins = { git = "https://github.com/veracruz-project/compiler-builtins.git", branch = "veracruz" }
```

Comment:
We no longer use Xargo to compile example. This file should be removed.

Reply:
Ah, can do